sporkfish/uci_client.py

```diff
@@ -236,7 +237,7 @@ def create_engine() -> Engine:
         ob = OpeningBook(
             OpeningBookConfig.from_dict(config.get("OpeningBookConfig"))  # type: ignore
         )
-        et = EndgameTablebase(
+        et = CompositeTablebase(
```

Comment:
Do we need to add to the config.yml file as well? Which db to use currently or the order